src/libraries/System.Diagnostics.Process/tests/ProcessModuleTests.Windows.cs

```diff
@@ -36,9 +36,7 @@ public void LongModuleFileNamesAreSupported()
             IntPtr moduleHandle = Interop.Kernel32.LoadLibrary(longNamePath);
             if (moduleHandle == IntPtr.Zero)
             {
-                Assert.Equal(126, Marshal.GetLastWin32Error()); // ERROR_MOD_NOT_FOUND
```

Comment:
could we bring back this assert in 7.0 and expand it to handle `122` (the error [in helix](https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-57551-merge-0bad4529cfd6489ebd/System.Diagnostics.Process.Tests/1/console.176badb8.log?sv=2019-07-07&se=2021-09-06T17:02:55Z&sr=c&sp=rl&sig=kky7AFZZa%2FGBIEFSw64gnh3dLNJy0djiLq89u5T1tSU%3D))?